cmake/modules/AddSwift.cmake

```diff
@@ -6,7 +6,7 @@ include(SwiftAndroidSupport)
 include(SwiftCXXUtils)
 
 function(_swift_gyb_target_sources target scope)
-  set(SWIFT_SYNTAX_GYB_SUPPORT_DIR "${SWIFT_PATH_TO_SWIFT_SYNTAX_SOURCE}/Sources/generate-swift-syntax-builder")
+  set(SWIFT_SYNTAX_GYB_SUPPORT_DIR "${SWIFT_PATH_TO_SWIFT_SYNTAX_SOURCE}")
```

Comment:
We don’t need this variable

Reply:
I thought that it might useful to have it in case we move `gyb_syntax_support` again. But we can probably just re-introduce it if needed. And hopefully we don’t move that directory anymore.